### ISSUE TITLE
MK220 Keyboard/mousecombo

### ISSUE THREAD

**Issue by Keeze** [pwr-Solaar/Solaar#152]
Hi Peter,

Any chance that Solaar is going to support the [MK220 combo](http://www.logitech.com/en-za/product/wireless-combo-mk220)?

That would be great!

Cheers,
Kees

It reports itself as: `Bus 002 Device 008: ID 046d:c534 Logitech, Inc.`

```
[ 3202.792902] usb 2-6.1: USB disconnect, device number 8
[ 3204.784350] usb 2-6.1: new full-speed USB device number 9 using ehci-pci
[ 3204.879094] usb 2-6.1: New USB device found, idVendor=046d, idProduct=c534
[ 3204.879099] usb 2-6.1: New USB device strings: Mfr=1, Product=2, SerialNumber=0
[ 3204.879104] usb 2-6.1: Product: USB Receiver
[ 3204.879107] usb 2-6.1: Manufacturer: Logitech
[ 3204.881303] input: Logitech USB Receiver as /devices/pci0000:00/0000:00:1d.7/usb2/2-6/2-6.1/2-6.1:1.0/input/input16
[ 3204.881421] hid-generic 0003:046D:C534.0007: input,hidraw0: USB HID v1.11 Keyboard [Logitech USB Receiver] on usb-0000:00:1d.7-6.1/input0
[ 3204.884764] input: Logitech USB Receiver as /devices/pci0000:00/0000:00:1d.7/usb2/2-6/2-6.1/2-6.1:1.1/input/input17
[ 3204.884996] hid-generic 0003:046D:C534.0008: input,hiddev0,hidraw3: USB HID v1.11 Mouse [Logitech USB Receiver] on usb-0000:00:1d.7-6.1/input1
```

**Comment by jrbenito** [comment]
Same as #222 and probably same as #234 

So, for #234 we shall have some progress based on the investigation done here.

**Comment by jrbenito** [comment]
This shall be fixed by PRs #337 and #346

**Comment by pfps** [comment]
There have been quite a few changes to solaar over the past few years so this issue may have already been addressed. If you are still using solaar and this issue has not been resolved please download and run the current version from this github repository. If the problem still persists add a new comment here, providing the output of `solaar -V` and `solaar show` and any new information about the issue.

If you are still running solaar you should be able to download and run its current version (in Linux) as follows:

1. Install git on your machine if it is not already installed. This is generally done via `sudo apt install git` or `sudo dnf install git`
2. Change to a directory where you want the code.
3. Download solaar via `git clone https://github.com/pwr-Solaar/Solaar.git`
4. Run solaar via `Solaar/bin/solaar`

For more information on installing solaar from github see https://github.com/pwr-Solaar/Solaar/blob/master/docs/installation.md
